src/composables/useTresContextProvider/index.ts

```diff
@@ -185,6 +193,31 @@ export function useTresContextProvider({
     root: ctx,
   }
 
+  // The loop
+
+  ctx.loop.register(() => {
```

Comment:
**question:** Can this be moved to `useRenderer`? This would be beneficial for the seperation of concerns and the dependencies would be more obvious.

Reply:
Ah, I had commented the opposite.

Moving this to the `useRenderer` means:

* it has to be passed `render`, `invalidate`, `advance`, and `renderMode` which it otherwise doesn't require
* it has to know and implement the update policies, which are orthogonal to "render this scene using this camera"

If we want to keep the update in `useRenderer`, I think it'd be better off encapulated in a passed `onRender` function that `useRenderer` can simply call. But I'd personally prefer that it be removed from `useRenderer` altogether. The render policy can be part of the loop:

```
if (renderPolicy.shouldRender()) {
    renderer.render()
}
```

Reply:
I already moved it.... can we please agree on one way @Tinoooo @andretchen0 ?

Reply:
@alvarosabu 

I'd prefer to reduce dependencies in `useRenderer`, but I won't block the PR because of it. It's your call.